### DISCUSSION TITLE
Is there any way to delete wiki.js at the bottom

### DISCUSSION THREAD

**Discussion by liuhuanshuo** [requarks/wiki#5884] [Help / Questions]
![](https://pic.liuzaoqi.com/picgo/202211231428512.png)I really like wiki.js

I want to use it for my company's technical team wiki

But now I'm having some problems and I don't want to show "Powered by Wiki.js" at the bottom.

I can't find where to set it. Can anyone help me

**Comment by cdn0x12** [comment]
There's no option available. Since wiki.js is open-source it can be done by modifying the code. However I think keeping the footer shows respect to the contributors of this project and does not affect your actual use.
